node/src/BaseClient.ts

```diff
@@ -4855,9 +4904,9 @@ export class BaseClient {
     /** Estimates the cardinality of the data stored in a HyperLogLog structure for a single key or
```

Comment:
While you are updating all commands, shall we make the style consistence with others?

```suggestion
    /** 
     * Estimates the cardinality of the data stored in a HyperLogLog structure for a single key or
```

Reply:
I try - although I don't prioritize style that doesn't affect the rendered product.  
So this change isn't really a must have.  I like to make it consistent for the developer's sake.